sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_client.py

```diff
@@ -377,6 +377,11 @@ def create_share(self, **kwargs):
         :keyword bool enable_snapshot_virtual_directory_access:
             Specifies whether the snapshot virtual directory should be accessible at the root of the share
             mount point when NFS is enabled. Default value is True.
+        :keyword bool paid_bursting_enabled: This property enables paid bursting.
+        :keyword int paid_bursting_max_bandwidth_mips: The maximum throughput the file share can support.
+            Current maximum for a file share is 10,340 MiB/sec.
```

Comment:
Let's remove the max from here, the keyword below, and everywhere else its used. We typically don't document things like this as they are server-side requirements that can change at any time and then we don't need to update the SDK every time they change.